.cicd/platforms/pinned/macos-10.14-pinned.sh

```diff
@@ -3,6 +3,8 @@ set -eo pipefail
 VERSION=1
 brew update
 brew install git cmake python libtool libusb graphviz automake wget gmp pkgconfig doxygen openssl@1.1 jq rabbitmq libpq postgres || :
+# install request and requests_unixsocket module
+pip3 install requests && pip3 install requests_unixsocket
```

Comment:
This can just be
`pip3 install requests requests_unixsocket`

Reply:
Done.